canvas/image.go

```diff
@@ -38,8 +48,9 @@ type Image struct {
 	Resource fyne.Resource // Load the image from an in-memory resource
 	Image    image.Image   // Specify a loaded image to use in this canvas object
 
-	Translucency float64   // Set a translucency value > 0.0 to fade the image
-	FillMode     ImageFill // Specify how the image should scale to fill or fit
+	Translucency float64    // Set a translucency value > 0.0 to fade the image
+	FillMode     ImageFill  // Specify how the image should scale to fill or fit
+	ScaleMode    ImageScale // set how the image is scaled
```

Comment:
nit: the comment should be capitalized, and maybe be a bit more descriptive. a user may ask what is the difference between "how the image should scale to fill or fit" and "how the image is scaled"

Reply:
**Specify how the image should be scaled** is ok?

Reply:
"Specify the type of scaling applied to the image content"?

Reply:
Hmm I think the dev could get confused by the two different uses of the word scale, but that may be more the fault of the FillMode comment above.
```suggestion
	FillMode	ImageFill // Specify how the image should expand to fill or fit the available space
	ScaleMode	ImageScale // Specify how type of scaling applied to the image
```